core/lib/spree/core/class_constantizer.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require 'active_support'
```

Comment:
Not sure to understand why it was working before this change, did you get that? Also, please use a permalink in the commit otherwise that file could change in the future and not point anymore to the right line.

Reply:
This worked because `active_support` was implicit required by `Spree::Core::Search::Base` when [spree/deprecation](https://github.com/solidusio/solidus/blob/ba78d7358c81baeeadab9a08d0bab9c21ea500ba/core/lib/spree/core/search/base.rb#L3) was required. I'll change the commit link. 👍

Reply:
Makes sense, thanks!